Source/fsConfig.cs

```diff
@@ -50,5 +50,10 @@ public static class fsConfig {
         /// Int64 and UInt64 will be serialized and deserialized as string for compatibility
         /// </summary>
         public static bool Serialize64BitIntegerAsString = false;
+
+        /// <summary>
+        /// Enums are serialized as string by default. Setting this to true will serialize them as integer.
```

Comment:
Can you replace this with:

Enums are serialized using their names by default. Setting this to true will serialize them as integers.